studies/models.py

```diff
@@ -515,6 +515,7 @@ class Documents(models.Model):
 
     informed_consent = models.FileField(
         _("Upload hier de toestemmingsverklaring (in .pdf of .doc(x)-formaat)"),
+        help_text=_("Bij algemeen belang, dien hier s.v.p. een leeg Word document in."),
```

Comment:
The English version is definitely not informative enough. I would argue the Dutch version could be more elaborate as well.

I would suggest: "When your lawful grounds for processing data is the Public Interest, for which no explicit consent is required, you may submit a blank document in this field."

And then something similar in Dutch.